test/Sandpit/Exscientia/Free_Energy/test_relative.py

```diff
@@ -0,0 +1,59 @@
+import pathlib
+
+import pytest
+import pandas as pd
+import numpy as np
+from alchemlyb.parsing.gmx import extract_u_nk
+
+import BioSimSpace.Sandpit.Exscientia as BSS
+from BioSimSpace.Sandpit.Exscientia.Protocol import FreeEnergyEquilibration
+from BioSimSpace.Sandpit.Exscientia.Align._decouple import decouple
+from BioSimSpace.Sandpit.Exscientia import Types as _Types
+
+class Test_gmx_ABFE():
+    @staticmethod
+    @pytest.fixture(scope='class')
+    def freenrg():
+        m = BSS.Parameters.openff_unconstrained_2_0_0("c1ccccc1").getMolecule()
+        decouple_m = decouple(m,
+                              property_map0={"charge": True, "LJ": True},
+                              property_map1={"charge": False, "LJ": False},
+                              intramol=False)
+        solvated = BSS.Solvent.tip3p(molecule=decouple_m,
```

Comment:
`BSS.Sovlent` requires that GROMACS is installed, so the test should also be skipped when it isn't installed. See [here](https://github.com/michellab/BioSimSpace/blob/devel/test/Process/test_gromacs.py) for an example.